packages/client/e2e/credit.spec.ts

```diff
@@ -49,6 +49,8 @@ test.describe("Credit", () => {
   })
 
   test.describe("New RFQ", () => {
+    test.describe.configure({ retries: 1 })
+    test.setTimeout(60_000)
```

Comment:
Extend test timeout only for this test

Reply:
you know what I am going to say 😉 
please add this to Timeout enum, with appropriate name - `EXTENDED` ? or `NEW_RFQ`, if it is really specialised just for this case .. this is an overall timeout for the whole test, right?